sdk/openai/Azure.AI.OpenAI/README.md

```diff
@@ -693,6 +693,22 @@ Console.WriteLine($"Translation ({translation.Duration.Value.TotalSeconds}s):");
 Console.WriteLine(translation.Text);
 ```
 
+### Generate Speech with Text-to-Speech (TTS) models
+
+```C# Snippet:SpeechGeneration
+SpeechGenerationOptions speechOptions = new()
+{
+    Input = "Hello World",
```

Comment:
(Totally optional, channeling some of the discussion from parallel conversations but can be deferred until later with no issues): if we'd like to, I think it's a defensible argument that `Input` on the new options type could be more descriptively named `Text`.

Reply:
Agreed, but I think we opted for `Input` to stay consistent with OpenAI. 

https://platform.openai.com/docs/guides/text-to-speech?lang=python

```
response = client.audio.speech.create(
  model="tts-1",
  voice="alloy",
  input="Today is a wonderful day to build something people love!"
)
```